<ui version="4.0" >
 <class>AServiceControlWidget</class>
 <widget class="QWidget" name="AServiceControlWidget" >
  <property name="modal" stdset="0" >
   <bool>false</bool>
  </property>
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>162</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>AIS Service Control</string>
  </property>
  <property name="sizeGripEnabled" stdset="0" >
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="upServiceNameLabel" >
       <property name="text" >
        <string>Service name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="upServiceNameLineEdit" />
     </item>
     <item row="0" column="2" >
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="upServiceStatusLabel" >
       <property name="text" >
        <string>Service status:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="upServiceStatusLineEdit" />
     </item>
     <item row="1" column="2" >
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="upStartupPathLabel" >
       <property name="text" >
        <string>Start-up file path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="upStartupPathLineEdit" />
     </item>
     <item>
      <widget class="QPushButton" name="upBrowseButton" >
       <property name="text" >
        <string>&amp;Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="upStartupTypeLabel" >
       <property name="text" >
        <string>Start-up type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="upStartupTypeComboBox" />
     </item>
     <item>
      <spacer name="horizontalSpacer_5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="upInstallButton" >
       <property name="text" >
        <string>&amp;Install</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upRemoveButton" >
       <property name="text" >
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upStartButton" >
       <property name="text" >
        <string>&amp;Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upStopButton" >
       <property name="text" >
        <string>S&amp;top</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
